src/battleground/Battleground.js

```diff
@@ -66,6 +73,10 @@ class Battleground extends React.Component<{||}> {
 		for (const t: Tank of this.testTanks) {
 			this.gameObjects.push(t);
 		}
+		//TODO: This needs to be put whereever health is updated
```

Comment:
You can just put this once at the end of battleground._rerender() and that should be fine